src/common/menu.rs

```diff
@@ -341,30 +351,32 @@ pub unsafe fn write_menu() {
     fs::write(path, data).unwrap();
 }
 
-pub unsafe fn spawn_menu() {
+pub unsafe fn spawn_menu() -> Result<(), Box<dyn std::error::Error>> {
     frame_counter::reset_frame_count(FRAME_COUNTER_INDEX);
     frame_counter::start_counting(FRAME_COUNTER_INDEX);
     let fname = "training_menu.html";
     let params = MENU.to_url_params();
+    let focus = FOCUS.lock()?;
     let page_response = Webpage::new()
         .background(Background::BlurredScreenshot)
         .htdocs_dir("contents")
         .boot_display(BootDisplay::BlurredScreenshot)
         .boot_icon(true)
-        .start_page(&format!("{}{}", fname, params))
+        .start_page(&format!("{}{}{}", fname, params, focus))
         .open()
         .unwrap();
 
-     let last_url = page_response
+    let last_url = page_response
         .get_last_url()
         .unwrap();
 
-    set_menu_from_url(last_url);
+    set_menu_from_url(last_url).unwrap();
```

Comment:
If we have are now returning a Result, maybe all of these unwrap()s can become `?`?